<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
					QWidget#bgwidget{

					background-image: url(C:/Projects/Python/Course/Course/assets/militarybcnd.png)
					}
				</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;; color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>С возвращением</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Выберите тип аккаунта</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="login_UAV">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>240</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>341</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>341</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
						border-radius:20px;
						background-color: rgb(255, 255, 255);
						font: 14pt &quot;MS Shell Dlg 2&quot;;
					</string>
    </property>
    <property name="text">
     <string>Войти как оператор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_admin">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>341</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>341</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
						border-radius:20px;
						background-color: rgb(255, 255, 255);
						font: 14pt &quot;MS Shell Dlg 2&quot;;
					</string>
    </property>
    <property name="text">
     <string>Войти как администратор</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>811</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="botlayout"/>
   </widget>
   <widget class="QPushButton" name="create">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>620</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>341</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>341</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
						border-radius:20px;
						background-color: rgb(255, 255, 255);
						font: 14pt &quot;MS Shell Dlg 2&quot;;
					</string>
    </property>
    <property name="text">
     <string>Зарегистироваться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>620</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>341</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>341</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
						border-radius:20px;
						background-color: rgb(255, 255, 255);
						font: 14pt &quot;MS Shell Dlg 2&quot;;
					</string>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>811</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="toplayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>811</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="testLayout"/>
   </widget>
   <widget class="QPushButton" name="test">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>600</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>341</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>341</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
						border-radius:20px;
						background-color: rgb(255, 255, 255);
						font: 14pt &quot;MS Shell Dlg 2&quot;;
					</string>
    </property>
    <property name="text">
     <string>Пробный запуск</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
